aws/resource_aws_cloudformation_stack_set.go

```diff
@@ -250,45 +250,54 @@ func resourceAwsCloudFormationStackSetUpdate(d *schema.ResourceData, meta interf
 func resourceAwsCloudFormationStackSetDelete(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).cfconn
 
-	input := &cloudformation.DeleteStackSetInput{
-		StackSetName: aws.String(d.Id()),
+	log.Printf("[DEBUG] Deleting CloudFormation StackSet: %s", d.Id())
+	err := deleteCloudFormationStackSet(conn, d.Id())
```

Comment:
These `delete*` functions will need some kind of identity object that can be generated from both the API object and a `schema.ResourceData`. The Stack Set case is simple, needing just the `stackSet.StackSetName`/`d.Id()`, but e.g. Stack Set Instance takes five identifying parameters and one operational parameter.

Reply:
These have been added